README.md

```diff
@@ -21,24 +21,15 @@ There is no web API to easily render complex layouts of text and other content i
-* The `CanvasRenderingContext2D.drawElementImage(element, x, y)` method renders `element` and its subtree into a 2D canvas at offset x and y, so long as `element` is a direct child of the `<canvas>`. It has no effect if `layoutsubtree` is not specified on the `<canvas>`.
-* The `WebGLRenderingContext.texElementImage2D(..., element)` method renders `element` into a WebGL texture. It has no effect if `layoutsubtree` is not specified on the `<canvas>`.
-
-(*) Without `layoutsubtree`, geometry APIs such as `getBoundingClientRect()` on these elements return an empty rect. They do have computed styles, however, and are keyboard-focusable.
-
-`drawElementImage(element ...)` takes the CTM (current transform matrix) of the canvas into consideration. Any drawn element is implicitly treated as having the CSS property `contain:paint`; overflowing content (including both layout and ink overflow) will be clipped to the element's content box. The optional `dwidth` and `dheight` arguments specify a destination rect in canvas coordinates to which the element will be scaled. If omitted, the element will have the same size and proportion when drawn to the canvas as it would have were it rendered outside the canvas.
-
-In addition, a `fireOnEveryPaint` option is added to `ResizeObserverOptions`, allowing script to be notified whenever any descendants of a `<canvas>` may render differently, so they can be redrawn. The callback to the resize observer will be called at resize observer timing, which is after DOM style and layout, but before paint.
-
-The same element may be drawn multiple times.
+* The `layoutsubtree` attribute on a `<canvas>` element opts-in canvas descendants to have layout and participate in hit testing. To aid in rendering, this attribute also causes the direct children of the `<canvas>` to have a stacking context, become a containing block for all descendants, and have paint containment.
+* The `CanvasRenderingContext2D.drawElementImage(element, x, y, width, height)` method renders `element` and its subtree into a 2D canvas at offset x and y. `layoutsubtree` must be specified on the `<canvas>`, and the `element` must be a direct child of the `<canvas>`. The current transform matrix of the canvas into consideration. Tranform properties on the element are ignored, but other effects are rendered, with overflowing content (including both layout and ink overflow) being clipped to the element's content box. The optional width/height arguments specify a destination rect in canvas coordinates to which the element will be scaled. If omitted, the element will have the same size and proportion when drawn to the canvas as it would have were it rendered outside the canvas. For WebGL/WebGPU, similar `WebGLRenderingContext.texElementImage2D` and `copyElementImageToTexture` functions are added.
```

Comment:
`The current transform matrix of the canvas into consideration` needs some grammatical love.

Reply:
And right after "Tranform properties" has an "s" missing in the "Transform".